src/atc/configurator/configurator.py

```diff
@@ -11,6 +10,8 @@
 
 from atc.exceptions import NoSuchValueException
 
```

Comment:
If the sql parser is dependent on a spark version - maybe we need to assert on it?
I guess some of the SQL syntax only applies for some versions.